config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   // DB
-  DATABASE_SERVER: 'localhost',
+  DATABASE_SERVER: '192.168.99.100',
```

Comment:
This is the IP of the docker virtual host. I tried to make it a little tidier by using the command `docker-machine ip default` but it was complaining because it didn't have the shell's context. I could probably fix this with a little more time but it appears to be static on my machine anyway. Would be interested to know if this works for you. If not I'll have another bash at it.